pkg/skaffold/build/local/jib_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local
+
+import (
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha3"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+	"testing"
+)
+
+func TestGenerateMavenCommand(t *testing.T) {
```

Comment:
FYI, tests that vary only inputs are often tested using test tables in Go. See https://github.com/golang/go/wiki/TableDrivenTests